stories/atoms/components/button/Button.stories.mdx

```diff
@@ -62,155 +67,204 @@ that will help users to find the action they're looking for.
-    )}
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        flexWrap: 'wrap',
+        gap: '50px'
+      }}
+    >
+      <Button backgroundColor="primary" variant="primary" label="Primary" size="medium" />
+      <Button backgroundColor="primary" variant="secondary" label="Secondary" size="medium" />
+      <Button
+        backgroundColor="primary"
+        variant="icon"
+        label="Icon"
+        size="medium"
+        icon={<Icon name="wallet" size={22} />}
+      />
+    </div>
   </Story>
 </Canvas>
 
 ## Sizes
 
-The `Button` component comes with three different sizes to allow its use according to its location within the interface.
+There are three different sizes to allow **Button** to be used according to its location within the interface.
 
-- **Small**: particularly suitable for small screens and must not saturate the space. This size should also be used for integrations in content spaces such as cards, footers, etc...
-- **Medium**: makes it possible to adapt to a greater number of uses.
-- **Large**: particularly suitable for large screens or to embed icons befor or after.
+- **Small**: particularly suitable for small screens and must not saturate the space. This size should also be used for integrations in content spaces such as cards, footers, etc.
+- **Medium** (_default_): makes it possible to adapt to a greater number of uses.
+- **Large**: particularly suitable for large screens or in conjonction with `leftIcon` / `rightIcon`.
 
 <Canvas>
   <Story name="Sizes">
-    {args => (
-      <div
-        style={{
-          display: 'flex',
-          alignItems: 'center',
-          justifyContent: 'center',
-          flexWrap: 'wrap',
-          gap: '25px'
-        }}
-      >
-        <Button backgroundColor="primary" variant="primary" label="Small" size="small" />
-        <Button backgroundColor="primary" variant="primary" label="Medium" size="medium" />
-        <Button backgroundColor="primary" variant="primary" label="Large" size="large" />
-      </div>
-    )}
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        flexWrap: 'wrap',
+        gap: '50px'
+      }}
+    >
+      <Button backgroundColor="primary" variant="primary" label="Small" size="small" />
+      <Button backgroundColor="primary" variant="primary" label="Medium" size="medium" />
+      <Button backgroundColor="primary" variant="primary" label="Large" size="large" />
+    </div>
   </Story>
 </Canvas>
 
 ## Background colors
 
-The background color of the `Button` component is declined according to the selected theme and allows to better mark a user action.
+**Button** have a palette of background colors to choose from and allows to better mark a user action.
+The background color can be a visual aid as to what type of action the button executes.
+
+By _default_ `backgroundColor` is **primary**.
```

Comment:
After reflection, I think we should not put "default" in italics, there are too many different texts for me. What do you think about it ?